rq/cli/cli.py

```diff
@@ -227,6 +227,11 @@ def worker(
     sentry_debug = sentry_debug or settings.get('SENTRY_DEBUG')
     sentry_dsn = sentry_dsn or settings.get('SENTRY_DSN')
     name = name or settings.get('NAME')
+    dict_config = options.get("dict_config", settings.get('DICT_CONFIG'))
+    
+    if dict_config:
+        import logging.config
+        logging.config.dictConfig(dict_config)
```

Comment:
Hmm this line is still marked as untested, which probably means this line is never triggered. Have you verified that this works properly? @juur